wadl/survey.py

```diff
@@ -229,6 +231,64 @@ def relink(self, routeParameters, write=True, showPlot=False):
         # call shutdowns and free stuff
         self.close()
 
+
```

Comment:
what is "completePct" ?

Reply:
percent of routes that will not complete (its just for simulating partial completion)